dashboard/test/models/script_test.rb

```diff
@@ -1754,13 +1754,13 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal ['English'], script.supported_locale_names
 
     script.supported_locales = ['fr-FR']
-    assert_equal ['English', 'French'], script.supported_locale_names
+    assert_equal ['English', 'Français'], script.supported_locale_names
 
     script.supported_locales = ['fr-FR', 'ar-SA']
-    assert_equal ['Arabic', 'English', 'French'], script.supported_locale_names
+    assert_equal ['English', 'Français', 'العربية'], script.supported_locale_names
 
     script.supported_locales = ['en-US', 'fr-FR', 'ar-SA']
-    assert_equal ['Arabic', 'English', 'French'], script.supported_locale_names
+    assert_equal ['English', 'Français', 'العربية'], script.supported_locale_names
```

Comment:
I'm trying to decide if we care about the ordering change here. If this was an interactable component I think we would, but given that it's display-only I don't think order really matters.

Reply:
Yeah that's my thought too. I guess the other thing is that this is sorted differently than the dropdown now.

Reply:
Yeah, I think _if_ we were going to have some opinionated sorting here we'd want it to mirror the dropdown

Reply:
I changed the function a little so that it roughly matches the order from before and does match the dropdown order.